### ISSUE TITLE
Expand to multiple references

### ISSUE THREAD

**Issue by jeromyanglim** [rmcgibbo/reftagger#6]
The common use case I find is where you have an existing article with manually entered references (perhaps in author-date format) and you want to convert the whole set to Bibtex (or any structured reference format for that matter).

Thus, it would be great if the system could parse multiple references at a time.

**Comment by rmcgibbo** [comment]
Yeah, that's a good idea. I wonder if it's permissible to just split the input on newlines and then run each split element through the tagger as-is, or if the chunking model needs to be more sophisticated.
